source/stylesheets/components/hero.styl

```diff
@@ -113,66 +126,36 @@
             margin-right: auto;
         }
     }
-
-    @media $media-query-append-retina-dpr {
-        .hero {
-            .berlin & {
-                background-image: url('/assets/images/berlin_hero_1920x1200.jpg');
-            }
-            .seattle & {
-                background-image: url('/assets/images/seattle_hero_1920x1200.jpg');
-            }
-        }
-    }
-
-    @media $media-query-append-retina-resolution {
-        .hero {
-            .berlin & {
-                background-image: url('/assets/images/berlin_hero_1920x1200.jpg');
-            }
-            .seattle & {
-                background-image: url('/assets/images/seattle_hero_1920x1200.jpg');
-            }
-        }
-    }
 }
 
 @media $media-query-960-and-up {
-    .hero {
-        background-size: auto auto;
+    $better-quality(1400);
+
+    .hero::before,
+    .hero::after {
+        background-size: 1400px 600px;
 
-        .berlin & {
-            background-image: url('/assets/images/berlin_hero_1400x600.jpg');
-        }
         .seattle & {
-            background-image: url('/assets/images/seattle_hero_1400x600.jpg');
             background-position: 50% 0;
         }
     }
+}
 
-    @media $media-query-append-retina-dpr {
-        .hero {
-            background-size: 1400px 600px;
+/*
```

Comment:
Should this block be uncommented, or removed? Or are you keeping it commented because you'll do more with this later?